lib/main/STM32G4/Middlewares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc.c

```diff
@@ -281,6 +281,7 @@ __ALIGN_BEGIN  uint8_t USBD_MSC_DeviceQualifierDesc[USB_LEN_DEV_QUALIFIER_DESC]
   * @}
   */
 
+static DMA_DATA_ZERO_INIT USBD_MSC_BOT_HandleTypeDef ClassData;
```

Comment:
Can you please put the library changes into a separate pull request - this will make it easier to find and re-apply them when the libraries are updated to a newer version in the future.

Reply:
Figured you'd request that, but this all in one place to start. See https://github.com/betaflight/betaflight/pull/10926.